### DISCUSSION TITLE
Mosaic Slider

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1189] [Proposals]
### Goal

Precisamos criar um slider semelhante ao atual com algumas configuração de grid com SnapCard ou PosterCard. 

![image](https://user-images.githubusercontent.com/123083886/232602517-2f7558d6-ff32-4cab-9ac1-6a0b6cdbefa4.png)

A lógica é semelhante ao do Carousel Slider, porém, contendo 6 coluna onde o Slide principal exibe em 5 colunas e o próximo exibe em 1 coluna a direita.

Existe a possibilidade de exibirmos 2 ou 3 blocos do Carousel. Podendo variar a quantidade de itens de acordo com suas configurações.

Exemplos:
![image](https://user-images.githubusercontent.com/123083886/232602960-0908e2c2-f4f9-4dbe-ac95-0aca39ef7a0b.png)

### Context

Estamos considerando a necessidade do novo layout da workCo onde teremos vários elementos distribuídos de formas diferentes que possam ser configuradas de acordo com a necessidade.

### Impact

Essa proposta dá liberdade de se criar inúmeras formas de exibição do componente e seus itens.

### Discussion Questions

- Existe algo mais que precisamos documentar? O Figma contém um arquivo piloto com um componente pronto para testes de variantes de exemplo.
![image](https://user-images.githubusercontent.com/123083886/232603630-ae3b71b3-fc7b-4eea-9dd2-350abddbbdc5.png)

### Do you have a visual proposal?

https://www.figma.com/file/j3f6uExXKSb8fPPiq9LdOC/Mosaic-Slider?node-id=1%3A2&t=sdPbi2Ht3gqxkdta-1

**Comment by yceballost** [comment]
Gracias Bruno, super buena información para evaluarlo con la definición del componente Grid

Detalles importantes es que el slider sería otra cosa diferente. Sería meter un mosaico dentro de un carrusel "libre". El espacio que aparece del siguiente item se gestionaría con el componente carrusel que eso tb deberíamos de evolucionarlo y habla de ello aquí https://github.com/Telefonica/mistica-design/discussions/979

Aún tenemos la duda de si será necesario un componente Mosaico o que el propio Grid pueda cubrir todos estos casos, todo esto lo resolveremos cuando trabajemos en el grid

**Reply by BrunoBiagioniNeto** [reply]
Acredito que com o Grid dentro do Carousel não seja necessário criar um Mosaico.

**Comment by yceballost** [comment]
Este componente de Mosaico se podrá resolver con el futuro componente Grid. Al necesitar tanta flexibilidad no vemos necesario generar un componente mosaico para tantos casos de uso y mejor solucionarlo con un componente más "primitivo" como el Grid.

Si en un futuro se detecta que se quiere acotar los usos del grid a casos muy concretos que se repitan en varios productos podría tener sentido un componente Mosaico que recoja los casos más generales.

Un pequeño ejemplo
https://tinyurl.com/2ggqymt4

cierro esta discussion
